### PR TITLE
feat: add SHIORI_PG_SSLMODE

### PR DESCRIPTION
**Pull request by refi64**
Some managed hosting providers don't support sslmode=disabled.

### CODE REVIEW

File: internal/cmd/root.go [hunk 1]
```diff
@@ -135,8 +135,12 @@ func openPostgreSQLDatabase(ctx context.Context) (database.DB, error) {
 	user, _ := os.LookupEnv("SHIORI_PG_USER")
 	password, _ := os.LookupEnv("SHIORI_PG_PASS")
 	dbName, _ := os.LookupEnv("SHIORI_PG_NAME")
+	sslmode, ok := os.LookupEnv("SHIORI_PG_SSLMODE")
```
Comment (fmartingr) [inline internal/cmd/root.go:138]: Keep in mind that in this case `ok` will be `true` in the case `SHIORI_PG_SSLMODE` is an empty string. Unsure how Posgres will react to that. Maybe is better to check for `sslMode == ""`.

**Review by fmartingr — APPROVED** [review]
Hey @refi64, thanks for the contribution! This looks good to go, though I have made a small comment on the flag check in case you want to clarify that for me. If you think that's good as it is, we can merge this into dev.

**Comment by AncientCatz** [comment]
When will this PR be merged into the master?

**Comment by fmartingr** [comment]
> When will this PR be merged into the master?

Waiting for [this comment](https://github.com/go-shiori/shiori/pull/536/files#r1032943673) to be addressed.

**Comment by fmartingr** [comment]
Thank you for your contribution!

**Pull request merged**